### PR TITLE
[chore] Modify readme for mill 0.2.3

### PR DESCRIPTION
**Pull request by franklinchou**
Mill 0.2.3 uses the following syntax for IntelliJ integration:
```
mill mill.scalalib.GenIdea/idea
```
See [here](https://github.com/lihaoyi/mill#intellij-setup).

### CODE REVIEW

**Comment by AlexITC** [comment]
When playsonify was released, it was built using `0.1.4`, it might be good to update to `0.2.3` as you propose, in this case, please update the [README.md](https://github.com/AlexITC/playsonify#development) file and the [travis config](https://github.com/AlexITC/playsonify/blob/master/.travis.yml) to specify that.

Thanks for your contribution!

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 before_install:
-- sudo curl -L -o /usr/local/bin/mill https://github.com/lihaoyi/mill/releases/download/0.2.2/0.2.2 && sudo chmod +x /usr/local/bin/mill
+- sudo curl -L -o /usr/local/bin/mill https://github.com/lihaoyi/mill/releases/download/0.2.3/0.2.3 && sudo chmod +x /usr/local/bin/mill
```
Comment (franklinchou) [inline .travis.yml]: @AlexITC Just wanted to point out that the travis script was actually running on mill `0.2.2`. Anyways, bumped version as requested.

Comment (AlexITC) [reply .travis.yml]: in fact it was running on `0.2.0`, it uses `0.2.2` in the `develop` branch.

File: build.sc [hunk 2]
```diff
@@ -34,7 +34,7 @@ trait PlaysonifyModule extends ScalaModule with PublishModule {
     organization = "com.alexitc",
     url = "https://github.com/AlexITC/playsonify",
     licenses = Seq(
-      License("MIT license", "https://github.com/AlexITC/playsonify/blob/master/LICENSE")
+      License.`MIT` // "https://github.com/AlexITC/playsonify/blob/master/LICENSE"
```
Comment (AlexITC) [inline build.sc]: why do we need this change?

**Comment by AlexITC** [comment]
Your changes have been applied on bd7c0cb655675, thanks!

**Pull request closed without merging**